### DISCUSSION TITLE
python virgin - how do I use this on win10?

### DISCUSSION THREAD

**Discussion by phil-barrett** [uPesy/easyeda2kicad.py#100] [General]
Nevermind.  figured it out.  

> Very excited about this!  But, I am utterly clueless.  Have py 3.5 installed.  Want to build a batch script.  Any help GREATLY appreciated.

**Comment by MostlyAcorn** [comment]
I failed with this too. I didn't have python so that installed from the Microsoft store, then I did pip install easyeda2kicad, but I could never find easyeda2kicad anywhere.  Moved to linux and it worked perfectly.

**Comment by donghoonpark** [comment]
Hi there 🤗
As a big fan of this library, I created a similar but more user-friendly GUI application for my colleague circuit engineers who are not comfortable with the command line.
I’d really appreciate your feedback and any suggestions for improvement.
Please check it out when you have some free time 😊
